UI/BuildDesignatorWnd.cpp

```diff
@@ -1051,7 +1051,8 @@ void BuildDesignatorWnd::CenterOnBuild(int queue_idx, bool open) {
             MapWnd* map = ClientUI::GetClientUI()->GetMapWnd();
             map->CenterOnObject(system_id);
             if (open) {
-                SelectSystem(system_id);
+                HumanClientApp::GetApp()->GetClientUI().GetMapWnd()->SelectSystem(system_id);
+                //SelectSystem(system_id);
```

Comment:
I'm not understanding your comment; I don't think I mistrust Git.

If you are asking about the commented-out line 1055, I simply forgot to remove it once I had line 1054 working right, and I guess I'll go ahead and add another commit here to remove it.

If you are asking why did I put this in as a PR rather than just pushing the commit to master and closing the related issue, it was to help make sure it was in the open long enough to get consideration as to whether this is enough of a bugfix to merit cherrypicking to 0.4.7 and therefor triggering an RC2 (I am slightly leaning towards thinking it is, particularly in light of the changelog edits Mat would also like to make).  It could be that just having the issue being flagged as optional for 0.4.7 was enough to ensure that consideration, but I wasn't sure.  Or perhaps there is some other mechanism it would have been better for me to rely on; pointers welcome.

If your question had some other focus, please clarify.